transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -188,7 +188,11 @@ private static void loadNativeLibrary() {
         if (!name.startsWith("linux")) {
             throw new IllegalStateException("Only supported on Linux");
         }
-        NativeLibraryLoader.load("netty-transport-native-epoll", PlatformDependent.getClassLoader(Native.class));
+        String []libraryNames = new String[]{
+          "netty-transport-native-epoll",
+          "netty_transport_native_epoll"
```

Comment:
> If you want to change that name too

Yes this was what I had in mind. We should have control over the library name and the library loading. I don't think we need to support loading mixed versions (older version libraries with the `-` in the names with newer version library loader which expects `_` in the name)  ... let me know if I'm missing something.